gbfs.md

```diff
@@ -840,6 +844,7 @@ Field Name | REQUIRED | Type | Defines
 \-&nbsp;`price` | Yes | Non-Negative float OR String | Fare price, in the unit specified by currency. If string, MUST be in decimal monetary value. <br/>*(added in v2.2)* Note: v3.0 may only allow non-negative float, therefore future implementations SHOULD be non-negative float.<br /><br />In case of non-rate price, this field is the total price. In case of rate price, this field is the base price that is charged only once per trip (e.g., price for unlocking) in addition to `per_km_pricing` and/or `per_min_pricing`.
 \-&nbsp;`is_taxable` | Yes | Boolean | Will additional tax be added to the base price?<br /><br />`true` - Yes.<br />  `false` - No.  <br /><br />`false` MAY be used to indicate that tax is not charged or that tax is included in the base price.
 \-&nbsp;`description` | Yes | String | Customer-readable description of the pricing plan. This SHOULD include the duration, price, conditions, etc. that the publisher would like users to see.
+\-&nbsp; `reserve_time` | OPTIONAL | Non-negative Integer | Maximum time in minutes that a vehicle can be reserved before a rental begins. This field SHOULD be used in cases where the ability to reserve a vehicle is dependant on the pricing plan. Where this is not the case `vehicle_types.json#reserve_time` SHOULD be used. When a vehicle is reserved by a user the vehicle remains locked until the rental begins. During this time the vehicle is unavailable and cannot be be reserved or rented by other users. The vehicle status in `free_bike_status.json` MUST be set to `is_reserved = true`. If the value of `reserve_time` elapses without a rental beginning, the vehicle status MUST change to `is_reserved = false`. If `reserve_time` is set to `0` vehicles cannot be reserved using this pricing plan. 
```

Comment:
Sorry, I find having reserve_time in two different feeds quite confusing. I don't want us having to implement a feature in two different ways because half of partners do it via pricing plans another half via vehicle types. Also, when you introduce default_pricing_plan_id in vehicle_types, you'll have to define which reserve time takes precedence (because a vehicle_type with reserve time A can reference default_pricing_plan with reserve time B). Since you already mention that this is a speculative use case, why not just remove it from the pricing plan feed? Other options: add guidance for which field should be used in which circumstances and perhaps rename the one in vehicle_types to default_reserve_time.

That said, we don't plan to use this field any time soon and I don't want to block the PR. Will just abstain from voting.